lib/doorkeeper/grape/helpers.rb

```diff
@@ -0,0 +1,40 @@
+require 'doorkeeper/grape/authorization_decorator'
+
+module Doorkeeper
+  module Grape
+    module Helpers
+      extend ::Grape::API::Helpers
+      include Doorkeeper::Rails::Helpers
+
+      # endpoint specific scopes > parameter scopes > default scopes
+      def doorkeeper_authorize!(*scopes)
```

Comment:
Assignment Branch Condition size for doorkeeper_authorize! is too high. [15.1/15]